lib/document.js

```diff
@@ -805,6 +805,14 @@ function init(self, obj, doc, opts, prefix) {
               reason: e
             }));
           }
+        } else if (opts.hydratedPopulatedDocs) {
```

Comment:
i just noticed the option `hydratedPopulatedDocs` is not documented in the `init` function here.
(also no options are documented in `Document.prototype.init` which is public)